app/components/ui/files/publish-modal/template.hbs

```diff
@@ -5,97 +5,82 @@
+    <div class="content">
+            <div class="ui basic segment">
+                <div class="ui list">
+                    <div class="item">
+                         <i class="environment folder icon"></i>
+                        <div class="content">
+                            <div class="environment name header">{{tale.title}}</div>
+                            <div class="list">
+                                {{#each nonOptionalFile as |file|}}
+                                <div class="item">
+                                        <label>
+                                            <div class="content">
+                                                <i class="file icon"></i>
+                                                 {{file}} <i class="info circle blue icon {{file}}"></i> 
+                                            </div>
+                                        </label>
                                     </div>
-                                </label>
+                                {{/each}}
                             </div>
-                        {{/each}}
+                        </div>
                     </div>
                 </div>
             </div>
         </div>
+    {{#if showPublishing}}
+    {{!-- Accordion that shows publishing status --}}
+    <hr class="publish-hr">
+    <div class="title">
+        <h2>
+            <i class="fas fa-briefcase"></i> Publishing
+            <span>
+                View your Tale's publishing status
+                <i class="dropdown icon"></i>
+            </span>
+        </h2>
     </div>
-</div>
-
-<hr>
-{{!-- Accordion menu that lets the user select a license --}}
-<div class="title">
-    <h2>
-        <i class="fab fa-creative-commons"></i> Usage Rights
-        <span>
-            Choose how you wish your Tale to be stored and reused
-            <i class="dropdown icon"></i>
-        </span>
 
-    </h2>
-</div>
-
-<div class="content">
-    <div class="ui form">
-        <div class="grouped fields">
-            {{#each availableLicenses as |license|}}
-            <div class="field">
-                <div class="ui radio checkbox {{license.short}}" id="{{license.spdx}}">
-                    <input type="radio" name="license-radio" checked="checked">
-                    <label><img class="ui small image left floated" src="{{license.imageName}}">
-                     {{license.name}} <i class="info circle blue icon {{license.short}}"></i></label> 
-                </div>
+    <div class="content">
+        <div class="ui form" style="height: 140px;">
+          <div class="progress-area">
+            {{#if publishing}}
+              <p class="publishing-now">Your Tale is being published to DataONE</p>
+            {{/if}}
+            <div class ="publish-progressbar">
+              {{ember-progress-bar progress=progress options=(hash strokeWidth=3 color=barColor trailColor= "#a5a4a4")}}
```

Comment:
Curious .. any major reason why you've opted for this third-party progress bar over using the built-in Semantic UI [Progress](https://semantic-ui.com/modules/progress.html) widget?

Reply:
I don't recall the exact reason why, but I originally wrote this using the semantic-ui progress bar and had issues getting it to work properly; I believe I was having issues controlling the state with handlebars while trying to avoid excessive jquery.

Reply:
As long as you tried the built-in one first ¯\_(ツ)_/¯

Seems like a perfectly valid reason to use a third-party component, imo